diff --git a/node_modules/@react-native-firebase/auth/lib/index.js b/node_modules/@react-native-firebase/auth/lib/index.js
index 1d7751c..b3da2b4 100644
--- a/node_modules/@react-native-firebase/auth/lib/index.js
+++ b/node_modules/@react-native-firebase/auth/lib/index.js
@@ -21,7 +21,7 @@ import {
   isNull,
   isString,
   isValidUrl,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 import { setReactNativeModule } from '@react-native-firebase/app/lib/internal/nativeModule';
 import {
   FirebaseModule,
