diff --git a/node_modules/@react-native-firebase/storage/app.plugin.js b/node_modules/@react-native-firebase/storage/app.plugin.js
new file mode 100644
index 0000000..c727814
--- /dev/null
+++ b/node_modules/@react-native-firebase/storage/app.plugin.js
@@ -0,0 +1,5 @@
+// node_modules/@react-native-firebase/storage/app.plugin.js
+const withDummyStoragePlugin = (config) => {
+    return config;
+  };
+  module.exports = withDummyStoragePlugin;
\ No newline at end of file
diff --git a/node_modules/@react-native-firebase/storage/lib/index.js b/node_modules/@react-native-firebase/storage/lib/index.js
index 2fecc42..c6aa279 100644
--- a/node_modules/@react-native-firebase/storage/lib/index.js
+++ b/node_modules/@react-native-firebase/storage/lib/index.js
@@ -15,7 +15,7 @@
  *
  */
 
-import { isAndroid, isNumber, isString } from '@react-native-firebase/app/lib/common';
+import { isAndroid, isNumber, isString } from '@react-native-firebase/app/lib/common/index.js';
 import { setReactNativeModule } from '@react-native-firebase/app/lib/internal/nativeModule';
 import {
   createModuleNamespace,
